posts/2020-09-10-survey-launch.md

```diff
@@ -0,0 +1,52 @@
+---
+layout: post
+title: "Launching the 2020 State of Rust Survey"
+author: The Rust Community Team
+description: "Hearing from you about the fifth year of Rust"
+---
+
+It's that time again! Time for us to take a look at how the Rust project is doing, and what we should plan for the future. The Rust Community Team is pleased to announce our [2020 State of Rust Survey][survey]! Whether or not you use Rust today, we want to know your opinions. Your responses will help the project understand its strengths and weaknesses and establish development priorities for the future. (If you'd like to give longer form feedback on the Rust roadmap, [we're also collecting blog posts!])
+
+Completing this survey should take about 10–15 minutes and is anonymous unless you choose to give us your contact information. We will be accepting submissions for the next two weeks (until September 24th), and we will write up our findings afterwards to [blog.rust-lang.org]. You can also check out [last year’s results][2019 survey].
+
+- [English][survey]
+- [Simplified Chinese]
+- [Traditional Chinese]
+- [French]
+- [German]
+- [Italian]
+- [Japanese]
+- [Polish]
+- [Portuguese]
+- [Russian]
+- [Spanish]
+- [Swedish]
+- [Vietnamese]
+
+(If you speak multiple languages, please pick one)
+
+Please help us spread the word by sharing the survey link on your social network feeds, at meetups, around your office, and in other communities.
+
+If you have any questions, please see our [frequently asked questions] or email the Rust Community team at [community-team@rust-lang.org].
+
+Finally, we wanted to thank everyone who helped develop, polish, and test the survey. In particular, we'd like to thank all of the volunteers who worked to provide all of the translations available this year and who will help to translate the results.
+
+[blog.rust-lang.org]: https://blog.rust-lang.org
+[frequently asked questions]: https://forge.rust-lang.org/community/survey-faq.html
+[community-team@rust-lang.org]: mailto:community-team@rust-lang.org
+[2019 survey]: https://blog.rust-lang.org/2020/04/17/Rust-survey-2019.htmlhttps://blog.rust-lang.org/2020/04/17/Rust-survey-2019.html
+
+[survey]: https://docs.google.com/forms/d/e/1FAIpQLSf__XKjS2xa55jUOi78ONvjG0elG5ZWqOz0MYdX6sgmcjb5pw/viewform?usp=sf_link
+[portuguese]: 
+[simplified chinese]: 
+[polish]: 
+[vietnamese]: 
+[french]: 
+[italian]: 
+[korean]: 
+[spanish]: 
+[russian]: 
+[traditional chinese]: 
+[swedish]: 
+[german]: 
+[japanese]: 
```

Comment:
These reference links won't match even once you add a URL, they need to start with an uppercase letter.